example/ideScripts/utilities.py

```diff
@@ -137,7 +161,12 @@ def verifyFolderStructure():
     buildDataPath = pathWithForwardSlashes(buildDataPath)
     # does not have backup file, always regenerated
 
-    vsCodeSettingsFolderPath = os.path.expandvars("%APPDATA%\\Code\\User\\")
+    # TODO make this not hard-coded

```

Comment:
Great note, I was also thinking about implementing default paths container somewhere.
I would add them to _templateStrings.py_.

Reply:
That sounds good! This way you could easily have a separate set of defaults for each system.